src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1

```diff
@@ -31,7 +31,90 @@ function Test-SimpleNewVm
         # Common
 		$x = New-AzureRmVM -Name $vmname -Credential $cred -DomainNameLabel $domainNameLabel
 
-        Assert-AreEqual $vmname $x.Name;        
+        Assert-AreEqual $vmname $x.Name;
+		Assert-Null $x.Identity 
+
+    }
+    finally
+    {
+        # Cleanup
+        Clean-ResourceGroup $vmname
+    }
+}
+
+<#
+.SYNOPSIS
+Test Simple Paremeter Set for New Vm with system assigned identity
+#>
+function Test-SimpleNewVmSystemAssignedIdentity
+{
+    # Setup
+    $vmname = Get-ResourceName
+
+    try
+    {
+        $username = "admin01"
+        $password = "werWER345#%^" | ConvertTo-SecureString -AsPlainText -Force
+        $cred = new-object -typename System.Management.Automation.PSCredential -argumentlist $username, $password
+		[string]$domainNameLabel = "$vmname-$vmname".tolower();
+
+        # Common
+		$x = New-AzureRmVM -Name $vmname -Credential $cred -DomainNameLabel $domainNameLabel -SystemAssignedIdentity
+
+        Assert-AreEqual $vmname $x.Name;
+		Assert-AreEqual "SystemAssigned" $x.Identity.Type     
+		Assert-NotNull  $x.Identity.PrincipalId
+		Assert-NotNull  $x.Identity.TenantId
+		Assert-Null $x.Identity.IdentityIds     
+    }
+    finally
+    {
+        # Cleanup
+        Clean-ResourceGroup $vmname
+    }
+}
+
+<#
+.SYNOPSIS
+Test Simple Paremeter Set for New Vm with user assigned identity
+#>
+function Test-SimpleNewVmUserAssignedIdentity
+{
+    # Setup
+    $vmname = "UAITG123456"
+
+    try
+    {
+       # To record this test run these commands first :
+       # New-AzureRmResourceGroup -Name UAITG123456 -Location 'Central US'
+       # New-AzureRmUserAssignedIdentity -ResourceGroupName  UAITG123456 -Name UAITG123456Identity
+       # 
+       # Now get the identity :
+       # 
+       # Get-AzureRmUserAssignedIdentity -ResourceGroupName UAITG123456 -Name UAITG123456Identity
+        #Nore down the Id and use it in the PS code
+		#$identityName = $vmname + "Identity1"
+		#$newUserIdentity =  New-AzureRmUserAssignedIdentity -ResourceGroupName $vmname -Name $identityName
+
+		#$newUserId = $newUserIdentity.Id
+
+		$newUserId = "/subscriptions/c9cbd920-c00c-427c-852b-8aaf38badaeb/resourcegroups/UAITG123456/providers/Microsoft.ManagedIdentity/userAssignedIdentities/UAITG123456Identity"
```

Comment:
@praries880 we should avoid using a specific subscription id -- anyone who doesn't have access to this subscription won't be able to re-record this test

Reply:
@cormacpayne This value is the Id of the user created identity, which the user will have to manually create following the steps in the comments above.

Reply:
Why can't we create the identity as part of the test?  Note that, it would be OK to simply mark this test as 'liveonly' for now, and file a test debt issue to add in the necessary setup after the release.

Reply:
Note that you could also test the correct creation of the identity as a unit test without having to have all the sceario test validations - I think a single scenario test that assigns both should be sufficient.